### ISSUE TITLE
Slicing for last n nodes is broken

### ISSUE THREAD

**Issue by vectorhacker** [graphql-python/graphene-mongo#20]
Using the slicing pagination for getting the first n items works, but asking for the last n doesn't. Doing this query for example works:
```graphql
{
  questions(first: 2) {
    edges {
      node{
        id
      }
      cursor
    }
  }
}
```

You get a result like this
```json
{
  "data": {
    "questions": {
      "edges": [
        {
          "node": {
            "id": "UXVlc3Rpb246NWFiNmM1OTY0YmFhM2MwMDA0YzIzMjBl"
          },
          "cursor": "YXJyYXljb25uZWN0aW9uOjA="
        },
        {
          "node": {
            "id": "UXVlc3Rpb246NWFiNmM1OTg0YmFhM2MwMDA0YzIzMjBm"
          },
          "cursor": "YXJyYXljb25uZWN0aW9uOjE="
        }
      ]
    }
  }
}
```

But doing this query breaks:

```graphql
{
  questions(last: 2) {
    edges {
      node {
        id
      }
      cursor
    }
  }
}
```

and give  you something like this

```json
{
  "errors": [
    {
      "message": "stop index must be greater than startindex for slice slice(None, -2, None)",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ]
    }
  ],
  "data": {
    "questions": null
  }
}
```

**Comment by abawchen** [comment]
@vectorhacker : Thanks for fixing!

**Comment by vectorhacker** [comment]
actually fixed in pull request #23
